app/src/shared/utils/confirm.jsx

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import React from 'react'
+import { render, unmountComponentAtNode } from 'react-dom'
+
+import ConfirmDialog, { type Properties } from '$shared/components/ConfirmDialog'
+
+const removeElementReconfirm = () => {
+    const target = document.getElementById('shared-confirm-alert')
+
+    if (target) {
+        unmountComponentAtNode(target)
+
+        if (target.parentNode) {
+            target.parentNode.removeChild(target)
+        }
+    }
+}
+
+const confirmDialog = (props: Properties): Promise<boolean> => new Promise((resolve: Function) => {
```

Comment:
Good point! Changed it now so that it just renders a dummy component and `Modal` does the mounting. Perhaps @mondoreale can comment if this a good way to do it?

Reply:
Yeah the way this is implemented is very unusual, this is basically what a react Portal does. e.g. @mondoreale's Modal uses a portal.

Reply:
I wanted to have `ConfirmDialog` appear using the existing `Modal` component but I couldn't find a better way to do it. I guess I could have my own `Portal`.